test/ui-testing/CustomInsert.test.js

```diff
@@ -0,0 +1,61 @@
+const timeout = process.env.SLOWMO ? 60000 : 10000;
+const fs = require('fs');
+beforeAll(async () => {
+	path = fs.realpathSync('file://../examples/index.html');
+	await page.goto('file://' + path, {waitUntil: 'domcontentloaded'});
+});
+
+describe('Custom Insert text', () => {
+	test('Add Custom Insert text in rich text mode', async () => {
+	await page.waitForSelector('.ple-module-body');
```

Comment:
I'd love to hear input from others on how this test looks. Any suggestions or proposed refinements? Thank you! @Shulammite-Aso @shreyaa-sharmaa @NitinBhasneria